### PR TITLE
Fix hover and focus state background in SocialButton

### PR DESCRIPTION
**Pull request by itscursedphil**

### CODE REVIEW

File: packages/components/src/SocialButton/SocialButton.tsx [hunk 1]
```diff
@@ -111,11 +111,19 @@ const socialButtonStyle = css<SocialButtonProps>`
...
-    border({ borderColor: 'background.secondary', theme })}
+    border({
+      borderColor: 'background.secondary',
+      theme
+    })}
 
-  &:hover,
-  &:focus {
+  &:hover :not(:disabled),
+  &:focus  &:focus :not(:disabled) {
```
Comment (johannessteu) [inline packages/components/src/SocialButton/SocialButton.tsx]:
```suggestion
  &:focus &:focus :not(:disabled) {
```

File: packages/components/src/SocialButton/SocialButton.tsx [hunk 2]
```diff
@@ -111,11 +111,19 @@ const socialButtonStyle = css<SocialButtonProps>`
...
-    border({ borderColor: 'background.secondary', theme })}
+    border({
+      borderColor: 'background.secondary',
+      theme
+    })}
 
-  &:hover,
-  &:focus {
+  &:hover :not(:disabled),
+  &:focus &:focus :not(:disabled) {
```
Comment (johannessteu) [inline packages/components/src/SocialButton/SocialButton.tsx]:
```suggestion
  &:focus, &:focus :not(:disabled) {
```

File: packages/components/src/SocialButton/SocialButton.tsx (hunk 1 above)
Comment (itscursedphil) [reply packages/components/src/SocialButton/SocialButton.tsx]: Good catch! 👀

File: packages/components/src/SocialButton/SocialButton.tsx (hunk 2 above)
Comment (itscursedphil) [inline packages/components/src/SocialButton/SocialButton.tsx]:
```suggestion
  &:focus :not(:disabled) {
```

**Pull request merged**